packages/payment-detection/test/erc20/fee-proxy-contract.test.ts

```diff
@@ -193,7 +193,7 @@ describe('api/erc20/fee-proxy-contract', () => {
         return Promise.resolve([
           // wrong fee address, ignore
```

Comment:
Maybe this comment can be edited, correct me if I'm wrong:
```suggestion
          // wrong fee address
```